articles/elemental.asm.xml

```diff
@@ -25,7 +25,11 @@
     </resource>
     </resources>
   <!-- Concept files -->
-    
+    <resources>
+      <resource xml:id="_elemental-about" href="../concepts/elemental-about.xml">
+        <description>About SLE Micro for Rancher</description>
```

Comment:
```suggestion
        <description>About &productnameshort;</description>
```
Then I'd keep it this way. FYI, this is just an internal title for the author, and it is not displayed anywhere. However, no problem to change it.